src/libraries/Microsoft.PowerFx.Interpreter/Environment/PowerFxConfigExtensions.cs

```diff
@@ -20,5 +20,15 @@ public static void AddFunction(this PowerFxConfig powerFxConfig, ReflectionFunct
         {
             powerFxConfig.AddFunction(function.GetTexlFunction());
         }
+
+        public static void AddDefaultEnums(this PowerFxConfig powerFxConfig)
+        {
+            powerFxConfig.EnumStoreBuilder.WithDefaultEnums();
```

Comment:
What's the difference between required/default?

Reply:
Default includes all defined enums. Required only pulls in the ones required for the current function list.

Reply:
I must say that's weird.  Generally, I would think that required means you must explicitly make an available choice; default is what you can get without making a choice if you're good with that.  

Is this a specialized meaning in the context of `powerFXConfig` ?

Reply:
So, in that case, Default is the set only used by PA-Client? That doesn't seem like something we need to expose publicly.

Reply:
I have reduced the scope of this change to only add the required enums in the public constructor of the PowerFxConfig.